sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/ConfigurationSettingSnapshot.java

```diff
@@ -86,12 +85,12 @@ public final class Snapshot {
     private String etag;
 
     /**
-     * Creates an instance of Snapshot class.
+     * Creates an instance of ConfigurationSettingSnapshot class.
      *
      * @param filters the filters value to set.
      */
-    @JsonCreator
-    public Snapshot(@JsonProperty(value = "filters", required = true) List<SnapshotSettingFilter> filters) {
+    public ConfigurationSettingSnapshot(
```

Comment:
I don't believe Jackson Databind will handle this correctly without the `JsonCreator` annotation